plonky3/Cargo.toml

```diff
@@ -13,31 +13,32 @@ rand = "0.8.5"
 powdr-analysis = { path = "../analysis" }
 powdr-executor = { path = "../executor" }
 
-p3-air = { git = "https://github.com/powdr-labs/Plonky3.git", branch = "main" }
-p3-matrix = { git = "https://github.com/powdr-labs/Plonky3.git", branch = "main" }
-p3-field = { git = "https://github.com/powdr-labs/Plonky3.git", branch = "main" }
-p3-uni-stark = { git = "https://github.com/powdr-labs/Plonky3.git", branch = "main" }
-p3-commit = { git = "https://github.com/powdr-labs/Plonky3.git", branch = "main", features = [
-  "test-utils",
+p3-air = { git = "https://github.com/powdr-labs/Plonky3.git", branch = "uni-stark-challenges-2" }
```

Comment:
is this just for tests or can we stay in main?

Reply:
There's a PR on our fork that needs to go in, then we can target main again

Reply:
Now pointing to main